docroot/themes/custom/mass_theme/templates/layout/html.html.twig

```diff
@@ -32,16 +32,101 @@
 
     <script>
       document.prefetchAlertsData = {};
+
+      // See: https://gist.github.com/nekman/297ebda63d6b00380058cbb0114296aa#file-polyfill-js-L586
```

Comment:
The polyfills look good and thanks for adding the comments! I wonder if we should move these custom polyfills into its own file and add it to mass_theme.libraries.yml or in alerts-block.js, instead of add them directly in the html.html.twig?

Reply:
per discussion with Luis, also these are IE11 polyfills for alerts-block but they also make sense to be in the global context

> ... if we need IE11 support and we need them as soon as possible, the answer is yes, there is no other way I know.
Anyhow, moving those polyfills makes sense, although on the other hand those polyfills are adding things in the global context.